assets/js/amp-post-meta-box.js

```diff
@@ -0,0 +1,57 @@
+/* exported AmpPostMetaBox */
+
+/**
+ * AMP Post Meta Box.
+ *
+ * @since 0.6
+ */
+var AmpPostMetaBox = ( function( $ ) {
+	'use strict';
+
+	// Exports.
+	return {
+		/**
+		 * Holds data.
+		 *
+		 * @since 0.6
+		 */
+		data: {},
+
+		/**
+		 * Boot plugin.
+		 *
+		 * @since 0.6
+		 * @param {Object} data Object data.
+		 * @return {void}
+		 */
+		boot: function( data ) {
```

Comment:
It seems overkill to have a `boot()` method just to call `addPreviewButton()` method but it is structured that way in preparation of #709.